src/tv2-common/inewsConversion/converters/ParseBody.ts

```diff
@@ -107,7 +107,8 @@ export type PartdefinitionTypes =
 	| Pick<PartDefinitionEkstern, 'type' | 'variant' | 'effekt' | 'transition'>
 	| Pick<PartDefinitionTelefon, 'type' | 'variant' | 'effekt' | 'transition'>
 
-const ACCEPTED_RED_TEXT = /\b(KAM(?:\d+)?|CAM(?:\d+)?|KAMERA(?:\d+)?|CAMERA(?:\d+)?|SERVER|ATTACK|TEKNIK|GRAFIK|EVS ?\d+(?:VOV?)?|VOV?|VOSB)+\b/gi
+const ACCEPTED_RED_TEXT = /\b(KAM(?:\d+)?|CAM(?:\d+)?|KAMERA(?:\d+)?|CAMERA(?:\d+)?|SERVER|ATTACK|TEKNIK|GRAFIK|EVS ?\d+ ?(?:VOV?)?|VOV?|VOSB)+\b/gi
+const EVS_RED_TEXT = /EVS ?(\d+) ?(VOV?)?/i
```

Comment:
Minor thing.
Since "Red text" is an iNews specific thing, maybe we don't need to mention red text i blueprints?
But then again, this file is in a folder called "inewsConversion", so I guess some iNews specific things are fine

Reply:
the whole thing is called sofie-blueprints-inews :)